src/Nethermind/Nethermind.TxPool.Test/TxPoolTests.Blobs.cs

```diff
@@ -1020,59 +1020,38 @@ private Transaction CreateBlobTx(PrivateKey sender, UInt256 nonce = default, int
-                    BlobSchedule = {
-                         initial with { Timestamp = _blockTree.Head.Timestamp  },
-                         updated with { Timestamp = _blockTree.Head.Timestamp + 1 },
-                    },
+                    IsEip4844Enabled = true,
+                    IsEip7594Enabled = true,
+                    MaxBlobCount = regularMaxBlobCount,
                 },
-                EngineChainSpecParametersProvider = Substitute.For<IChainSpecParametersProvider>()
-            });
+                ForkOnBlockNumber = _blockTree.Head.Number + 1,
+            };
 
             Block head = _blockTree.Head;
             _blockTree.FindBestSuggestedHeader().Returns(head.Header);
 
-            TxPoolConfig txPoolConfig = new() { BlobsSupport = BlobsSupportMode.InMemory, Size = 10 };
-            _txPool = CreatePool(txPoolConfig, provider);
+            _txPool = CreatePool(specProvider: provider);
             EnsureSenderBalance(TestItem.AddressA, UInt256.MaxValue);
 
-            UInt256 nonce = 0;
-
-            foreach (var blobCount in blobTxBlobCounts)
-            {
-                _txPool.SubmitTx(CreateBlobTx(TestItem.PrivateKeyA, nonce++, blobCount), TxHandlingOptions.None);
-            }
-
-            Assert.That(_txPool.GetPendingBlobTransactionsCount(), Is.EqualTo(blobTxBlobCounts.Length));
+            _txPool.SubmitTx(CreateBlobTx(TestItem.PrivateKeyA, 0, regularMaxBlobCount), TxHandlingOptions.None);
+            Assert.That(_txPool.GetPendingBlobTransactionsCount(), Is.EqualTo(1));
 
             await AddBlock();
 
-            return _txPool.GetPendingBlobTransactionsCount();
-        }
-
-        public static IEnumerable EvictionAccordingToLimitsTestCaseSource
-        {
-            get
-            {
-                static TestCaseData MakeTestCase(string testName, BlobScheduleSettings initial, BlobScheduleSettings updated, int[] blobTxBlobCounts, int finalTxCount)
-                    => new(initial, updated, blobTxBlobCounts) { TestName = $"EvictionAccordingToBlobLimits: {testName}", ExpectedResult = finalTxCount };
-
-                yield return MakeTestCase("Evicts when per block limit is set",
-                    new BlobScheduleSettings { Max = 6 }, new BlobScheduleSettings { Max = 5 }, [6], 0);
-                yield return MakeTestCase("Evicts when per tx limit only is set",
-                    new BlobScheduleSettings { Max = 6 }, new BlobScheduleSettings { Max = 6, MaxBlobsPerTx = 3 }, [6], 0);
-                yield return MakeTestCase("Evicts when per tx limit only is changed",
-                    new BlobScheduleSettings { Max = 6, MaxBlobsPerTx = 3 }, new BlobScheduleSettings { Max = 6, MaxBlobsPerTx = 2 }, [2, 3, 2], 1);
-                yield return MakeTestCase("Evicts next txs too",
-                    new BlobScheduleSettings { Max = 6 }, new BlobScheduleSettings { Max = 6, MaxBlobsPerTx = 3 }, [6, 3, 3, 3], 0);
-            }
```

Comment:
What happened with all the previous test cases? No test had `blobTxBlobCounts = []` while the new test only checks this case.